### ISSUE TITLE
Cannot execute AUTO mission more than once during single flight.

### ISSUE THREAD

**Issue by nikhil9** [ArduPilot/ardupilot#776]
AUTO mission script is executed only once during flight. If Auto mode is switched for second time, RTL is executed and plane loiters around home. I am using mission planner 1.2.92 with Arduplane 2.75
Don't know if it is bug or code is made that way but it will be nice to have capability to execute AUTO mission more than once during single flight especially when you abort Autoland.
